### DISCUSSION TITLE
Clarify usage and access conditions, use standard clauses

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#33] [General]
Many countries do not have public access to Inspire data, for example Germany. Work should be done to get more open licenses for this data.

Fees required to access the data, and the unclear licensing terms create obstacles. To avoid barriers in the usage of INSPIRE datasets, the existing fees should be waived. The use of standard licenses (e.g., CC By) to inform about conditions for access and use is recommended too

**Comment by thorsten-reitz** [comment]
In The ff.aii use case, we have encountered that data is in principle accessible, e.g. on the forest inventory, but no access and use conditions/limitations are defined, and the data custodian is not able or willing to cleary define these.

**Comment by AndersOstmanNovogit** [comment]
The first part of the issue (public access) is similar to issue #32 and described in section 4.2.4 in D3.4.

The second issue (missing licensing conditions) is described in section 4.2.6 in D3.4.
@MayteTDGeograma and @thorsten-reitz : Which solutions did you apply? Gentle(wo)mens agreement?

**Reply by AndersOstmanNovogit** [reply]
Now reported in D4.3 section 4.3.3

**Comment by MayteTDGeograma** [comment]
It was not an agreement between Gentle(wo)mens :-)
To make sure, we searched for popImpact data from countries that did have defined access and no fees.
